src/coord/src/catalog.rs

```diff
@@ -732,18 +771,23 @@ impl Catalog {
                 schema_name: String,
             },
             DropItem(GlobalId),
+            UpdateItem {
+                id: GlobalId,
+                name: FullName,
+                item: CatalogItem,
+            },
         }
 
         let temporary_ids = self.temporary_ids(&ops)?;
         let mut actions = Vec::with_capacity(ops.len());
         let mut storage = self.storage();
         let mut tx = storage.transaction()?;
         for op in ops {
-            actions.push(match op {
-                Op::CreateDatabase { name } => Action::CreateDatabase {
+            let mut ops = match op {
```

Comment:
`ops` is a bit confusing here because the result of this transformation is actions, not ops. But you can avoid having to come up with a name by just wrapping the map in `actions.extend(...)`, analogously to how it was wrapped in `actions.push(...)` before.